myhpom/templates/myhpom/accounts/signup.html

```diff
@@ -20,68 +20,95 @@ <h5 class="pseudo-modal__title--centered">Already have an account? <a href="{% u
+                            <input type="text" class="form-control{% if form.errors.middle_name %} is-invalid{% endif %}" name="middle_name" id="id_middle_name" value="{% if form.data.middle_name %}{{form.data.middle_name}}{% endif %}" />
+                            <div class="invalid-feedback">
+                                {% if form.errors.middle_name %}{{ form.errors.middle_name }}{% endif %}
+                            </div>
                         </div>
                         <div class="col-md-6 col-lg-4 form-group">
-                            <label for="id_last_name">Last name</label>
-                            <input type="text" class="form-control" name="last_name" id="id_last_name" />
+                            <label for="id_last_name">{{form.fields.last_name.label|upper}}</label>
+                            <input type="text" class="form-control{% if form.errors.last_name %} is-invalid{% endif %}" name="last_name" id="id_last_name" value="{% if form.data.last_name %}{{form.data.last_name}}{% endif %}" />
+                            <div class="invalid-feedback">
+                                {% if form.errors.last_name %}{{ form.errors.last_name }}{% endif %}
+                            </div>
                         </div>
                     </div>
                     <div class="row">
                         <div class="col-lg-8">
                             <div class="row">
                                 <div class="col-md-6 form-group">
-                                    <label for="id_email_address">Email address</label>
-                                    <input type="email" class="form-control is-invalid" name="email_address" id="id_email_address" />
+                                    <label for="id_email">{{form.fields.email.label|upper}}</label>
+                                    <input type="email" class="form-control{% if form.errors.email %} is-invalid{% endif %}" name="email" id="id_email" value="{% if form.data.email %}{{form.data.email}}{% endif %}" />
                                     <div class="invalid-feedback">
-                                        Please enter a valid email address.
+                                        {% if form.errors.email %}{{ form.errors.email }}{% endif %}
                                     </div>
                                 </div>
                                 <div class="col-md-6 form-group">
                                     <label for="id_state">State of residence</label>
-                                    <select name="state" id="id_state" class="form-control">
-                                        <option value="NC">North Carolina</option>
-                                        <option value="SC">South Carolina</option>
+                                    <select name="state" id="id_state" class="form-control{% if form.errors.state %} is-invalid{% endif %}">
```

Comment:
As we're using bs4 I'm tempted to sugged we used the dropdown support and put a divider between supported states and the other states: http://getbootstrap.com/docs/4.1/components/dropdowns/

Reply:
I would like to defer this change.